app/models/event.rb

```diff
@@ -0,0 +1,22 @@
+class Event < ApplicationRecord
+  has_many :event_members
+  has_many :users, through: :event_members
+
+  def to_icon(event_number:)
+    {
+      id: id,
+      event_number: event_number,
+      start_date: start_date,
+      end_date: end_date,
+      image_url: image_url,
+      name: name,
+      members: users.map(&:to_icon)
+    }
+  end
+
+  def add_members(members:)
```

Comment:
Right now this is only used in specs.  But I'm thinking I would keep it in the model because at some point we'll need this.